src/libs/ajax.js

```diff
@@ -1041,38 +1042,15 @@ const Workspaces = signal => ({
   }
 })
 
+const Catalog = signal => ({
+  getDatasets: async () => {
+    const res = await fetchCatalog('v1/datasets', _.merge(authOpts(), { signal }))
+    return res.json()
+  }
+})
 
-const DataRepo = signal => ({
-  getSnapshots: async () => {
```

Comment:
This was now unused, so I just pulled it. I am unsure the value of the hard coded extra fields though.

Reply:
I'm also unsure, but from #2714 and #2868, they seem like hard-coded development/demo data. #2714 says the attributes added there would be reverted "once the beta test cycle is complete". The attributes added in #2868 look like they were just for the purposes of development/review and possibly should never have been committed at all.